hooks/UseLogin.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { RegisterProps } from "../types";
+
+export const UseLogin = () => {
+  const router = useRouter();
+  const [errorText, setErrorText] = useState("");
+
+  const login = ({ email, password }: RegisterProps) => {
+    try {
+      axios
+        .post(`${process.env.url}/login`, {
+          email: email,
+          password: password,
+        })
+        .then((res) => {
+          if (res.status === 200 || 201) {
```

Comment:
it doesn't work as you expect) This statement is always `true` since `!!201` is `true`